### PR TITLE
Update to frontend 1.0.0

### PR DESCRIPTION
**Pull request by alex-ju**
This PR:
- updates the Design System to use [govuk-frontend 1.0.0](https://github.com/alphagov/govuk-frontend/releases/tag/v1.0.0) 🎉
- adds the tab component example and guidance
- removes the container class from the header examples (as now the container class is being added by default)
- removes the `npm login` instructions for the Design System

[Trello card](https://trello.com/c/noTIT53J)

### CODE REVIEW

File: src/components/tabs/default/index.njk [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Tabs
+layout: layout-example.njk
+---
+
+{% from 'tabs/macro.njk' import govukTabs %}
+
+{{ govukTabs({
+  "items": [
```
Comment (nickcolley) [inline src/components/tabs/default/index.njk]: "items" => items

File: src/components/tabs/default/index.njk [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "label": "Past day",
+      "id": "past-day",
+      "panel": {
+        "html": "<h2 class=\"govuk-heading-l\">Past day</h2>\n<table class=\"govuk-table\">\n  <thead class=\"govuk-table__head\">\n    <tr class=\"govuk-table__row\">\n      <th class=\"govuk-table__header\" scope=\"col\">Case manager</th>\n      <th class=\"govuk-table__header\" scope=\"col\">Cases opened</th>\n      <th class=\"govuk-table__header\" scope=\"col\">Cases closed</th>\n    </tr>\n  </thead>\n  <tbody class=\"govuk-table__body\">\n    <tr class=\"govuk-table__row\">\n      <td class=\"govuk-table__cell\">David Francis</td>\n      <td class=\"govuk-table__cell\">3</td>\n      <td class=\"govuk-table__cell\">0</td>\n    </tr>\n    <tr class=\"govuk-table__row\">\n      <td class=\"govuk-table__cell\">Paul Farmer</td>\n      <td class=\"govuk-table__cell\">1</td>\n      <td class=\"govuk-table__cell\">0</td>\n    </tr>\n    <tr class=\"govuk-table__row\">\n      <td class=\"govuk-table__cell\">Rita Patel</td>\n      <td class=\"govuk-table__cell\">2</td>\n      <td class=\"govuk-table__cell\">0</td>\n    </tr>\n  </tbody>\n</table>\n"
```
Comment (nickcolley) [inline src/components/tabs/default/index.njk]: We can use 'capturing' here which is loads safer...

See the conditional Nunjucks https://deploy-preview-369--govuk-design-system-preview.netlify.com/components/radios/

File: src/components/tabs/index.md.njk [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+Test your content without tabs first. Consider if it’s better to:
+
+- simplify and reduce the amount of content
+- split the content across multiple pages
+- keep the content on a single page, separated by headings
+- use a table of contents to let users navigate quickly to specific sections of content
+
+## How it works
+
+There are 2 ways to use the tabs component. You can use HTML or, if you’re using [Nunjucks](https://mozilla.github.io/nunjucks/) or the [GOV.UK Prototype Kit](https://govuk-prototype-kit-beta.herokuapp.com/), you can use the Nunjucks macro.
```
Comment (nickcolley) [inline src/components/tabs/index.md.njk]: govuk-prototype-kit-beta -> govuk-prototype-kit

**Comment by igloosi** [comment]
remove npm login done here #368

**Comment by alex-ju** [comment]
@nickcolley: updated addressing all three comments

**Comment by alex-ju** [comment]
@igloosi: I guess this is the downside of not working on cards or issues 😄

**Pull request merged**